src/clj/harja/palvelin/integraatiot/tloik/tekstiviesti.clj

```diff
@@ -158,7 +167,15 @@
               paikankuvaus
               tr-osoite
               selitteet
-              lisatietoja))))
+              lisatietoja
+              viestinumero
```

Comment:
Miksi tämä on näin monta kertaa?

Reply:
Ilmoitusviesti konstruioidaan formatterilla, missä jokainen kuittaustyyppi saa saman viestinnumeron etuliikkeekseen, esim A12, V12, jne. 
![image](https://user-images.githubusercontent.com/261821/221811098-4b04e571-9041-460f-8c4c-157dc8d28bd1.png)

Reply:
Näyttää sitten lopputuloksena tältä: 
![image](https://user-images.githubusercontent.com/261821/221811409-e982a783-c59e-46ee-92a1-ab26d42639b4.png)

Reply:
Aivan, käy järkeen :+1: